### PR TITLE
ATT-314: Load all connection requests

### PR DESCRIPTION
**Pull request by altairrostam**
[ATT-314](https://myaria-health.atlassian.net/browse/ATT-314?atlOrigin=eyJpIjoiYWNhYTIyNWFiNzQzNGJmOTg2ZTZhNWVjYzkxMzBlOWEiLCJwIjoiaiJ9)

Child issue of [ATT-181](https://myaria-health.atlassian.net/browse/ATT-181?atlOrigin=eyJpIjoiMmRhY2FlMmY2M2ZkNDI3OTk2OGE1ZjBmYWE5ZmY1OGIiLCJwIjoiaiJ9)

### CODE REVIEW

File: app/(protected)/dashboard/(admin)/patient/requests/page.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+						pagination
+						disableSelectionOnClick
+					/>
+				</ShowIf>
+			</Box>
+		</Box>
+	);
+}
+
+export default PatientRequests;
```
Comment (wilsonangara) [inline app/(protected)/dashboard/(admin)/patient/requests/page.tsx:151]: nit: add end of file line

File: app/(protected)/dashboard/(admin)/patient/requests/page.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
+"use client"
+import { useGetPatientConnectionsByStatus } from "@api/connection/connection";
+
+import { useDashboard } from "@commons/context";
+import { ShowIf } from "@components/layouts";
+
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import Stack from "@mui/material/Stack";
```
Comment (wilsonangara) [inline app/(protected)/dashboard/(admin)/patient/requests/page.tsx:9]: combine these to 1 import

File: app/(protected)/dashboard/(admin)/patient/requests/page.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+import { DataGrid } from "@mui/x-data-grid/DataGrid";
+
+import PowerIcon from "@mui/icons-material/Power";
+import VisibilityIcon from '@mui/icons-material/Visibility';
+
+import { GridCellParams, GridColDef } from "@mui/x-data-grid";
```
Comment (wilsonangara) [inline app/(protected)/dashboard/(admin)/patient/requests/page.tsx:15]: Combine import `DataGrid` with `GridCellParams` and `GridColDef`

**Review by wilsonangara — APPROVED** [review]
Other than that, LGTM

File: app/(protected)/dashboard/(admin)/patient/requests/page.tsx (hunk 3 above)
Comment (altairrostam) [reply app/(protected)/dashboard/(admin)/patient/requests/page.tsx:15]: nice catch. thanks

**Pull request merged**